drracket-test/tests/drracket/private/easter-egg-lib.rkt

```diff
@@ -64,7 +64,7 @@ and then loading the framework after that.
        (define drr-frame (wait-for-drracket-frame))
        (set-module-language! drr-frame)
        (queue-callback/res
-        (λ () (send (send (send drr-frame get-definitions-text) get-canvas) focus)))
+        (λ () (send+ drr-frame (get-definitions-text) (get-canvas) (focus))))
```

Comment:
I don't think this one is a good change, actually. It isn't like the example in the documentation for `send+` where we're getting objects of the same type back and doing functional update.